src/main/scala/model/world/disturbingFactors/Factors.scala

```diff
@@ -0,0 +1,86 @@
+package model.world.disturbingFactors
+
+import model.genome.Alleles.AlleleKind
+import model.genome.Genes.GeneKind
+import model.world.Climate
+import model.world.Generation.Population
+
+import scala.util.Random
+
+trait Factor {
+  import model.world.disturbingFactors.FactorTypes._
+
+  /** @return the normal damage that is applied with this Factor */
+  val normalDamage: Double
+
+  /**
+   * Method that applies the damage of this Factor on a Population with the given climate
+   * @param bunnies
+   *   the actual Population
+   * @param climate
+   *   the actual Climate
+   * @return
+   *   the remaining Population
+   */
+  def applyDamage(bunnies: Population, climate: Climate): Population
+
+  /** @return the FactorType of this Factor */
+  def factorType: FactorKind
+}
+
+trait FactorWithOneGene extends BasicFactor {
```

Comment:
Da quel che ho capito qui si intendono i fattori che agiscono sui coniglietti in base al "valore" (Allele) che assume un unico Gene, se ho capito bene allora questo lo chiamerei `FactorOnSingleGene` e l'altro `FactorOnDoubleGene`, il 'with' mi manda un po' in confusione